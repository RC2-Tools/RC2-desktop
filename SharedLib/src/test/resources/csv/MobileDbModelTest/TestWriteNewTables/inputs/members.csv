beneficiary_entity_row_id,beneficiary_entity_status,date_created,member_id,custom_member_form_id,custom_member_row_id,status,status_reason
1,,2017-09-11T22:29:13.907000000,1,,,ENABLED,
1,,2017-09-11T21:39:29.442000000,2,,,ENABLED,
1,,2017-09-11T22:31:58.571000000,3,,,DISABLED,Married into another family
2,,2017-09-11T22:56:18.502000000,1,,,ENABLED,
3,,2017-09-11T22:56:18.502000000,5,,,ENABLED,
