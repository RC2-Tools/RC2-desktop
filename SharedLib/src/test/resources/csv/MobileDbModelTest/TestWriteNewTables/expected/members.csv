"_id","_form_id","_locale","_savepoint_type","_savepoint_timestamp","_savepoint_creator","_create_user","_last_update_user","_deleted","_data_etag_at_modification","_row_etag","beneficiary_entity_row_id","beneficiary_entity_status","custom_member_form_id","custom_member_row_id","date_created","_default_access","_group_modify","_group_privileged","_group_read_only","_row_owner","member_id","status","status_reason"
,,,,,,,,false,,,"1","","","","2017-09-11T22:29:13.907000000",,,,,,"1","ENABLED",""
,,,,,,,,false,,,"1","","","","2017-09-11T21:39:29.442000000",,,,,,"2","ENABLED",""
,,,,,,,,false,,,"1","","","","2017-09-11T22:31:58.571000000",,,,,,"3","DISABLED","Married into another family"
,,,,,,,,false,,,"2","","","","2017-09-11T22:56:18.502000000",,,,,,"1","ENABLED",""
,,,,,,,,false,,,"3","","","","2017-09-11T22:56:18.502000000",,,,,,"5","ENABLED",""
